tajo-cli/src/main/java/org/apache/tajo/cli/tsql/DefaultTajoCliOutputFormatter.java

```diff
@@ -206,4 +278,33 @@ public static String parseErrorMessage(String message) {
 
     return message;
   }
+
+  /**
+   * borrowed from Presto
+   */
+  private static boolean detectRealTerminal()
```

Comment:
My previous question was too short, so you may miss my point. 
I mean, why are the comments in Presto's detectRealTerminal() excluded when you borrow?
Please see https://github.com/facebook/presto/blob/master/presto-cli/src/main/java/com/facebook/presto/cli/ConsolePrinter.java. It will be helpful for us as well.